src/Main.ts

```diff
@@ -408,16 +410,22 @@ export class Main {
             name: METRIC_PUPPETS,
         });
         const bridgeBlocked = prometheus.addGauge({
-            name: "bridge_blocked",
+            name: "blocked",
```

Comment:
Is that the bug? What's wrong with `bridge_blocked`?

Reply:
`prometheus.addGauge` prefix's with bridge_ by default, so you end up with bridge_bridge_blocked :p

my bad for not catching the first time.

Reply:
Oh on /o\ Good catch!